not/localization/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The localization module provides localized user-facing strings
+
+generally, one need only import the polyglot singleton object
+
+This module uses an unorthodox approach to singletons to avoid keeping
+state in the module, which feels more awkward.
+
+When python 3.9 comes out in earnest, revisit using:
+https://www.python.org/dev/peps/pep-0603/
+"""
+
+import ctypes
+from json import load
+from locale import getdefaultlocale, normalize, windows_locale
+from os import getenv
+from pathlib import Path
+from platform import system
+from typing import Dict, Union
+
+DEFAULT_LOCALE = normalize("en")
+STRINGS_SUPPORTED = set("")
+
+
+class _Polyglot(Dict[str, str]):
+    """This class is responsible for lazy-loading the best-effort translation of user-facing
+    strings."""
+
+    language_alias_map = {
+        DEFAULT_LOCALE: {"en", "en_US"},
+    }
+    language_file_map = {
+        DEFAULT_LOCALE: "en.json",
+    }
+
+    def __init__(self) -> None:
+        system_locale = get_system_locale()
+        self.locale = next(
+            (
+                k
+                for k, v in self.language_alias_map.items()
+                if system_locale == k or system_locale in v
+            ),
+            DEFAULT_LOCALE,
+        )
```

Comment:
i'm a bit of of a names-over-nesting-guy. a NONG, if u will.

would you be open to something along the lines of:

```python
maybe_locale_from_alias_map = (
    k for k, v in self.language_alias_map.items() 
    if system_locale == k or system_locale in v
)
self.locale = next(maybe_locale_from_alias_map, DEFAULT_LOCALE)
```

?

Reply:
Nooooormaly dont love assigning single-use variables. But I have nothing against your laws, sheriff. I am a visitor in this town, and as long as my name is Smeke the 🐍 you know ill oblige.